hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/common/HoodieFlinkEngineContext.java

```diff
@@ -166,17 +167,17 @@ public void setJobStatus(String activeModule, String activityDescription) {
   }
 
   @Override
-  public void putCachedDataIds(String basePath, String instantTime, int... ids) {
+  public void putCachedDataIds(HoodieDataCacheKey cacheKey, int... ids) {
```

Comment:
may be in a follow up patch. 
why can't we support persisting and unpersisting for other engines too?